source/slang/slang-check-inheritance.cpp

```
@@ -8,6 +8,134 @@
+{
+    for (auto const& stamp : entry.dependencyEpochs)
+    {
+        if (getDeclExtensionEpoch(stamp.decl) != stamp.epoch)
+            return false;
+    }
+
+    return true;
+}
+
+void SharedSemanticsContext::_collectInheritanceInfoDependencyEpochs(
+    Decl* subjectDecl,
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)